src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/commands.py

```diff
@@ -102,7 +102,7 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
         g.storage_command_oauth('list', 'list_blobs', transform=transform_storage_list_output,
                                 table_transformer=transform_blob_output)
         g.storage_command_oauth('download', 'get_blob_to_path', table_transformer=transform_blob_output)
-        g.storage_command_oauth('generate-sas', 'generate_blob_shared_access_signature')
+        g.storage_custom_command_oauth('generate-sas', 'generate_sas_blob_uri', transform=transform_url)
```

Comment:
Since you do the transformation in your custom command, you shouldn't need `transform=transform_url`

Reply:
Actually, this makes sense to have.